src/backend/commands/explain.c

```diff
@@ -1859,7 +1859,7 @@ ExplainNode(PlanState *planstate, List *ancestors,
 
 	if (ResManagerPrintOperatorMemoryLimits())
 	{
-		ExplainPropertyInteger("operatorMem", PlanStateOperatorMemKB(planstate), es);
+		ExplainPropertyLong("operatorMem", PlanStateOperatorMemKB(planstate), es);
```

Comment:
why not https://github.com/arenadata/gpdb/commit/f3e4b95edb6cbf0cd41dc9ff0c884bfdc425e6fb?

Reply:
I'm not sure how good an idea this is, since `ExplainPropertyLong` is only used once in this patch.

Reply:
that commit change `ExplainPropertyInteger` to accept a 64bit argument

Reply:
Won't this break ABI?